app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -299,6 +299,19 @@ class FileManagerFile extends FileRenderers.RawTableFile {
       icon = <i className="fa fa-file-o" aria-hidden="true" />;
     }
 
+    let download_icon = (
+      <svg
+        className="submission-download"
```

Comment:
The file manager is actually used in other places beyond submissions (for example, assignment starter files). So we should keep the naming generic: something like `file-download`, for example.